instrumentation/grpc-1.6/library/src/test/java/io/opentelemetry/instrumentation/grpc/v1_6/propagator/GrpcPropagator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.grpc.v1_6.propagator;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapGetter;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.context.propagation.TextMapSetter;
+import java.util.Collections;
+import java.util.List;
+
+public class GrpcPropagator implements TextMapPropagator {
+
+  public static final String FIELD = "X-grpc-field";
+  private static final String METADATA = "value";
+
+  @Override
+  public List<String> fields() {
+    return Collections.singletonList(FIELD);
```

Comment:
Suggest to use static import here, for more style guidance you can refer to 
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports

Reply:
helpful suggestions